src/support_sphere/lib/auth/login/login.dart

```diff
@@ -0,0 +1,147 @@
+import 'package:ionicons/ionicons.dart';
+import 'package:loading_overlay/loading_overlay.dart';
+import 'package:flutter/material.dart';
+import 'package:support_sphere/components/icon_logo.dart';
+import 'package:support_sphere/components/text_form_builder.dart';
+import 'package:support_sphere/components/password_form_builder.dart';
+import 'package:support_sphere/auth/register/register.dart';
+
+class Login extends StatefulWidget {
+  const Login({Key? key}) : super(key: key);
+
+  @override
+  _LoginState createState() => _LoginState();
+}
+
+class _LoginState extends State<Login> {
+  @override
+  Widget build(BuildContext context) {
+    // LoginViewModel viewModel = Provider.of<LoginViewModel>(context);
```

Comment:
Add a todo for when we're gonna use this maybe? (& other comments)

or remove, up to you :)